### PR TITLE
Fix assigned cursor

### LINKED ISSUE #1193: `className` is not assigned to `Cursor` in chart
### Which package has an issue?

@yamada-ui/react

### Package Version

1.3.3

### Description

`use-chart-tooltip.ts`
```diff ts
      cursor: {
-        className: cursorClassName,
+        className: cx("ui-chart__cursor", cursorClassName),
      },
```

### Link to Reproduction

_No response_

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by xBlackTea**
Closes #1193

## Description

className is  assigned to Cursor in chart

## New behavior
className is  assigned to Cursor
## Is this a breaking change (Yes/No):
No
## Additional Information

### CODE REVIEW

**Review by 108yen — APPROVED** [review]
Thank you!

File: .changeset/eight-kangaroos-bathe.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/charts": patch
+---
+
+fix: assigned-cursor
```
Comment (108yen) [inline .changeset/eight-kangaroos-bathe.md:5]: Can you change the description of `changeset`?
```diff md
-fix: assigned-cursor
+fix: assigned className to cursor
```

**Review by 108yen — CHANGES_REQUESTED** [review]

File: .changeset/eight-kangaroos-bathe.md (hunk 1 above)
Comment (xBlackTea) [reply .changeset/eight-kangaroos-bathe.md]: i did it.

**Review by 108yen — APPROVED** [review]

File: .changeset/eight-kangaroos-bathe.md (hunk 1 above)
Comment (108yen) [reply .changeset/eight-kangaroos-bathe.md]: Thank you!!

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**